docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/radio/radio.scss

```diff
@@ -5,11 +5,39 @@
 @import '../../00-base/mixins/form';
 
 .civictheme-radio {
+  $root: &;
+
   position: absolute;
+  display: flex;
+  align-items: center;
 
   .civictheme-radio__element {
     @include civictheme-checkbox-radio('radio');
```

Comment:
why do we need this mixin instead of just having rules here?

Reply:
Do we need to separate both component CSS without using mixins?